community/code-of-conduct.md

```diff
@@ -0,0 +1,97 @@
+# Music Encoding Initiative Community Code of Conduct
+
+## **1. Purpose**
+
+A primary goal of the Music Encoding Initiative community is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe, and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
+
+This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
+
+We invite all those who participate in the Music Encoding Initiative community to help us create safe and positive experiences for everyone.
+
+## **2. Open-Source Community Citizenship**
+
+A supplemental goal of this Code of Conduct is to increase open-source community citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community.
+
+Communities mirror the societies in which they exist, and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
+
+If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to fully contribute, we want to know.
+
+## **3. Expected Behavior**
+
+The following behaviors are expected and requested of all community members:
+
+* Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+
+* Exercise consideration and respect in your speech and actions.
+
+* Attempt collaboration before conflict.
+
+* Refrain from demeaning, discriminatory, or harassing behavior and speech.
+
+* Be mindful of your surroundings and of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+
+* Remember that community event venues may be shared with members of the public; please be respectful to all patrons of these locations.
```

Comment:
The empty lines between the list bullets are not needed.  The resulting HTML list is valid. Nevertheless, the list entries will each be wrapped in an HTML paragraph element which will take up more space. If that is your intention, it is fine with me, but usually, we have more compact lists on the website (accomplished by leaving away the additional line break).

```suggestion
* Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
* Exercise consideration and respect in your speech and actions.
* Attempt collaboration before conflict.
* Refrain from demeaning, discriminatory, or harassing behavior and speech.
* Be mindful of your surroundings and of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
* Remember that community event venues may be shared with members of the public; please be respectful to all patrons of these locations.
```

You can compare the result in the following example:

* this is a list with

* paragraphs

* caused by the additional line-breaks

vs.

* a more
* compact list
* without additional empty lines